data/json/items/armor.json

```diff
@@ -4452,7 +4452,7 @@
     "id": "helmet_nomad",
     "type": "ARMOR",
     "name": "nomad cowl",
-    "description": "A makeshift cowl with protection for the eyes and a peak or visor, designed for long travels.",
+    "description": "A makeshift cowl with protection for the eyes from rain and sun, designed for long travels.",
```

Comment:
Why is "and a peak or visor" removed?

Reply:
"and a peak or visor" doesn't really make sense, nor is it useful description. The peak of the cowl would be the top of the "hood" component, and the visor would be the brim/bill of the baseball hat component. The nomad cowl has both, not just one, and the materials you use to create this (baseball cap) and the name of the item (cowl) already imply this. It's just confusing and useless filler, and the description doesn't state anywhere that the cowl provides both glare protection and is waterproof, which are important details, and the mail reason you make it.